docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -152,105 +157,143 @@ which are based on the 2013 WHO guidelines on management of SAM in children and
 
-.. _waste_tx2.1:
 
-In infants 0-6 months
-+++++++++++++++++++++
+Severe acute malnutrition (SAM)
++++++++++++++++++++++++++++++++
+
+SAM in infants (0-6 months)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 :underline:`Classify SAM in infants`
 
 * Any grade of bilateral pitting oedema (+, ++ or +++) OR
 * WHZ < -3 zscore 
 
-**Treatment**
+Treatment of infants with SAM in the SC
 
 NOTE: All infants 0-6 months of age with SAM with or without medical complications should be referred to the SC.
 
-:underline:`Classify MAM in infants`
-
-* MUAC of lactatating mother of infant 0-6 months <23.0 cm
-* WHZ ≥-3 to <-2 AND 
-  * No bilateral pitting oedema AND 
-  * No medical complications
-  * Clinically well and alert
-
-**Treatment** 
-
-Assess the infant’s feeding and counsel the mother or caregiver on appropriate IYCF practices. Emphasize on establishing effective exclusive
-breastfeeding. If feeding problems, follow up in 5 days. If no feeding problem, follow up in 30 days. Admit the mother to TSFP. Refer the mother for Productive Safety Net Programme (PSNP).
 
-:underline:`Classify no acute malnutrition in infants`
+SAM in children (>6 months)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* WHZ ≥-2 zscores AND 
-* No bilateral pitting oedema
+Since 2007, a new model of care called community-based therapetic care (CTC) or community-based management of acute malnutrition (CMAM) has been endorsed for the treatment of acute malnutrition which addresses the limitations of previous inpatient therapeutic feeding programmes. Patients with severe malnutrition, with good appetite and without medical complications are treated in the outpatient therapeutic programme (OTP) that provides ready-to-use therapeutic food (RUTF) and medicines to treat simple conditions. The food and medicines are taken home and patient attends OTP site weekly for monitoring and resupply. 
 
-**Treatment** 
-
-Congratulate and counsel the mother on appropriate IYCF practices.
+Severely malnourished persons with medical complications and/or anorexia are treated in an inpatient stabilization center (SC) where they receive standard World Health Organization (WHO)- recommended initial care until they have enough appetite and are well enough to continue with outpatient care. Approximately 10-15% of children in CMAM programmes require inpatient care at the stabilisation centers and the average length-of-stay at an SC is 14 days before they are transferred out to an OTP. [Scott_2020]_ [Tekeste_2012]_ 
 
-.. todo::
-    What about MAM with oedema? Are they treated as SAM or MAM?
-    Answer: they are treated as MAM in GBD, but in reality, probably SAM. Mam with oedema is approximately 2% of MAM. 
+:underline:`Classify SAM without medical complications`
 
-.. _waste_tx2.2:
+* Bilateral pitting oedema + or ++ OR 
+* WHZ <-3 z-scores AND 
+  * Appetite test passed
+  * No medical complications
+  * Clinically well and alert
```

Comment:
Your indentation is off here, so the last 3 bullets are showing up in line with the previous bullet rather than as separate bulleted items.